test/unit_tests/cvode/CXX_serial/cv_test_getjac.cpp

```diff
@@ -168,6 +170,35 @@ int main(int argc, char* argv[])
   // SUNDIALS context object for this simulation
   sundials::Context sunctx;
 
+  // Comparison tolerance
+  sunrealtype tol = 100 * std::sqrt(SUN_UNIT_ROUNDOFF);
+  if (argc > 1)
+  {
+#if defined(SUNDIALS_SINGLE_PRECISION)
+    tol = std::stof(argv[1]);
+#elif defined(SUNDIALS_DOUBLE_PRECISION)
+    tol = std::stod(argv[1]);
+#elif defined(SUNDIALS_EXTENDED_PRECISION)
+    tol = std::stold(argv[1]);
+#else
+#error "SUNDIALS precision macro not defined"
+#endif
+    if (tol <= ZERO)
+    {
+      std::cerr << "ERROR: Invalid tolerance, tol = " << tol << std::endl;
+      return 1;
+    }
+  }
+
+  // Integration tolerances
+#if defined(SUNDIALS_EXTENDED_PRECISION)
+  const sunrealtype rtol = SUN_RCONST(1.0e-8);
+  const sunrealtype atol = SUN_RCONST(1.0e-12);
+#else
+  const sunrealtype rtol = SUN_RCONST(1.0e-6);
```

Comment:
Are these tolerances too tight for single precision?

Reply:
Yes, although the CI passes currently if we ran for longer this might become a problem